packages/docusaurus/src/webpack/utils.ts

```diff
@@ -489,43 +448,30 @@ export function getMinimizer(useSimpleCssMinifier = false): Plugin[] {
         },
       },
     }),
-  ];
-
-  if (useSimpleCssMinifier) {
-    minimizer.push(
-      new OptimizeCSSAssetsPlugin({
-        cssProcessorPluginOptions: {
-          preset: 'default',
-        },
-      }),
-    );
-  } else {
-    minimizer.push(
-      ...[
-        new OptimizeCSSAssetsPlugin({
-          cssProcessorPluginOptions: {
-            preset: CssNanoPreset,
-          },
-        }),
-        new OptimizeCSSAssetsPlugin({
-          cssProcessor: CleanCss,
-          cssProcessorOptions: {
-            inline: false,
-            level: {
-              1: {
-                all: false,
-              },
-              2: {
-                all: true,
-                restructureRules: true,
-                removeUnusedAtRules: false,
-              },
+    new CssMinimizerPlugin({
+      minimizerOptions: {
+        // note: require -> import throws
+        preset: () => require('@docusaurus/cssnano-preset'),
+      },
+    }),
+    new CssMinimizerPlugin({
+      minimizerOptions: {
+        minify: async (data, inputMap) => {
```

Comment:
@RDIL could you please add the options for CleanCSS that used before?

```js
inline: false,
level: {
  1: {
    all: false,
  },
  2: {
    all: true,
    restructureRules: true,
    removeUnusedAtRules: false,
  },
```

And wonder why did you enable sourcemap option?

Reply:
Oh, I didn't see this!  Source maps should only be for dev (things like the Sources tab), which I'll fix.  Same with adding back the options.